addons/contexts/src/preview/libs/getContextNodes.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Test on the merging result of a pair of settings', () => {
   it('should correctly merge two settings', () => {
     // setup
     const someTopLevelSettings = {
-      icon: 'box',
+      icon: 'box' as const,
```

Comment:
Why casting it const instead of using a readonly type? :thinking:

Reply:
Without it `someTopLevelSettings.icon` is typed to `string` but with 'as const' it is typed as "box"

Reply:
This feature is really useful (added recently in TS 3.4). Previously I can not named a variable but have to pass in directly due to the reason that @gaetanmaisse explained.

Because of that I previous type `icon` as `string`, but `Icons` component is not happy about it so I have also declare an empty namespace then I loose what that @gaetanmaisse just made for us... but not anymore!